### ISSUE TITLE
Issues about method of runing

### ISSUE THREAD

**Issue by Escapekk** [pyeprog/peng#1]
I completed the compilation according to the instructions in readme.md. Two executable files, tests and app, are generated in the build/bin directory, but I don’t know how to use them correctly.

**Comment by Soda100Percent** [comment]
I have the same issue...

**Comment by niuniu666-ignore** [comment]
So, the problem has been solved?

**Comment by niuniu666-ignore** [comment]
So, the problem has been solved?

**Comment by Soda100Percent** [comment]
> So, the problem has been solved?

It seems like an empty project, except that the libs are included.

**Comment by niuniu666-ignore** [comment]
yes, i check the code using related artical, the details of code are empty.



---- Replied Message ----
| From | ***@***.***> |
| Date | 09/15/2024 09:04 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [pyeprog/peng] Issues about method of runing (#1) |

So, the problem has been solved?

It seems like an empty project, except that the libs are included.
